code/game/machinery/doors/airlock.dm

```diff
@@ -2001,11 +2009,15 @@
 	name = "glass airlock"
 	opacity = FALSE
 	glass = TRUE
+	greyscale_config = /datum/greyscale_config/airlocks/window
```

Comment:
are you sure this isn't a failure of this block of code?
![image](https://github.com/user-attachments/assets/3154578b-1c36-4b43-962e-2d28a3b161b1)
also unrelated do you know what this block of code is sposed to do

Reply:
Looks like something I wrote 3 years ago, I'm somewhat confident this was probably contributing to the issue but altogether I'd rather adjust it so that this *only* adjusts trim-strings.

Reading my own mind here, I'm guessing the thought was that we would assign the grayscale config in init as opposed to on subtypes, but like... in hindsight I'd rather assign this on the subtype for legibility.